admin/data_tools/text_edition_cleanup.ipynb

```diff
@@ -0,0 +1,493 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%load_ext autoreload\n",
+    "%autoreload 2"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os, sys, re\n",
+    "import json\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "#add parent dir to module search path to find readQueryCursor module\n",
+    "sys.path.insert(0,os.path.dirname(os.getcwd()))\n",
+    "#print(sys.path)\n",
+    "import readQueryCursor as rqc\n",
+    "\n",
+    "\n",
+    "dbname = 'homer3'\n",
+    "\n",
+    "conf = rqc.readConnConfig\n",
+    "conf['database'] = dbname\n",
+    "conf['user'] = 'postgres'\n",
+    "conf['password'] = 'gandhari'\n",
```

Comment:
```suggestion
    "from dotenv import load_dotenv\n",
    "\n",
    "\n",
    "load_dotenv()\n",
    "\n",
    "dbname = 'homer3'\n",
    "\n",
    "conf = rqc.readConnConfig\n",
    "conf['database'] = dbname\n",
    "conf['user'] = os.getenv('DB_USERNAME')\n",
    "conf['password'] = os.getenv('DB_PASSWORD')\n",
```

Reply:
did you change requirements.txt to include dotenv?

Reply:
I have, in addition to pandas(to read csv file), requests(http request for the image), pillow(to open the image and get the size) and livereload(for web development, a server which in our case, helps to always get the last updated file from the file directory, not from the cached version stored in browser).

Yes, I also think if we import the .env file to docker-compose.yml, we would not have to use load_dotenv() to load in the Python environment. (I would have to check this)

Reply:
Anyone using the code locally would have to ensure that the environment gets loaded or add the information directly.